DEMO/LEDsetup.py

```diff
@@ -1,21 +1,16 @@
-
-# coding: utf-8
```

Comment:
UTF-8 is the default, so there's no need to put this in.  It's very rare you actually use this - by convention, everyone programs in UTF-8.  Only if you had embedded characters in some alternate encoding would you use this.